### PR TITLE
Digitization stream split

### PR DESCRIPTION
**Pull request by brownd1978**
Split digitization stream into functional output files.  This also simplifies the NoPrimary mixing workflow by treating NoPrimary generation as a 'normal' primary.

### CODE REVIEW

File: JobConfig/mixing/epilog.fcl [hunk 1]
```diff
@@ -1,38 +1,36 @@
...
-    @table::Digitize.TriggeredOutput
-    fileName: "dig.owner.MixTriggered.version.sequencer.art"
-  }
-  UntriggeredOutput : {
-    @table::Digitize.UntriggeredOutput
-    fileName: "dig.owner.MixUntriggered.version.sequencer.art"
-  }
-}
+outputs : @local::Digitize.Outputs
+#  SignalOutput :  {
```
Comment (soleti) [inline JobConfig/mixing/epilog.fcl:23]: Is there a reason why you need to keep these lines commented out?

**Comment by brownd1978** [comment]
@FNALbuild run build test with Mu2e/Offline #761

File: JobConfig/mixing/epilog.fcl (hunk 1 above)
Comment (brownd1978) [reply JobConfig/mixing/epilog.fcl]: No reason.  I removed them

**Comment by brownd1978** [comment]
@FNALbuild run build test with Mu2e/Offline #761

**Comment by brownd1978** [comment]
@FNALbuild run build test with Mu2e/Offline #761

**Comment by kutschke** [comment]
@soleti FYI: Dave says that he has finished his planned work on this PR.  Please have a look and let us know if you have additional requests; if not please approve.

**Review by soleti — APPROVED** [review]

**Pull request merged**